feature-libs/cart/base/assets/translations/en/cart.i18n.ts

```diff
@@ -76,4 +76,12 @@ export const cart = {
       'The product configuration requires additional entries in the back end. As a result, you cannot proceed. Please contact support.',
     inProgress: 'Processing',
   },
+  cartToast: {
+    entriesAdded: '{{ quantity }} items added',
```

Comment:
There seems to be a mechanism available to manage plurals.  Look for i18n keys that have the `_other`  suffix.  (I think is used to be `_plural` before.)